.gitignore

```diff
@@ -11,3 +11,5 @@ pom.xml
 *#
 .#*
 */.lein-repl-history
+*/out/
+.nrepl-port
```

Comment:
Can you please remove this from the pull request, as it's not part of this specific problem?